src/test/java/com/example/resource/MovieIT.java

```diff
@@ -207,6 +208,75 @@ void shouldReturnNotFoundWhenUpdatingNonExistingMovie() {
 
     }
 
+    @Test
+    @DisplayName("Delete should return status 200")
+    void deleteShouldReturnStatus200() {
+
+        UUID uuid = UUID.randomUUID();
+        Movie movie = createMovie(uuid, "frank Zappa", "Horror", 3.3f, 1985, "Friday the 13:th");
+        String requestBody = convertToJson(movie);
+
+        RestAssured.given()
+            .contentType(MediaType.APPLICATION_JSON)
+            .body(requestBody).when()
+            .post("/movies/")
+            .then()
+            .statusCode(201);
+
+        RestAssured.given()
+            .when()
+            .delete("/movies/" + movie.getUuid())
+            .then()
+            .statusCode(200 )
+            .body(equalTo("Successfully deleted"));
+
+    }
+
+    @Test
+    @DisplayName("Delete without uuid should return 405")
+    void deleteWithoutUuidShouldReturn405() {
+        UUID uuid = UUID.randomUUID();
+        Movie movie = createMovie(uuid, "frank Zappa", "Horror", 3.3f, 1985, "Friday the 13:th");
+        String requestBody = convertToJson(movie);
+
+        RestAssured.given()
+            .contentType(MediaType.APPLICATION_JSON)
+            .body(requestBody).when()
+            .post("/movies/")
+            .then()
+            .statusCode(201);
+
+        RestAssured.given()
+            .when()
+            .delete("/movies/" )
+            .then()
+            .statusCode(405 );
+
+    }
+
+    @Test
+    @DisplayName("Delete with wrong uuid should return 404")
+    void deleteWithWrongUuidShouldReturn404() {
+        UUID uuid = UUID.randomUUID();
+        Movie movie = createMovie(uuid, "frank Zappa", "Horror", 3.3f, 1985, "Friday the 13:th");
+        String requestBody = convertToJson(movie);
+
+        RestAssured.given()
+            .contentType(MediaType.APPLICATION_JSON)
+            .body(requestBody).when()
+            .post("/movies/")
+            .then()
+            .statusCode(201);
+
```

Comment:
I dont think we need to create a movie in this test? should be enough with the secod part, generating a uuid and trying to delete